css/styles.css

```diff
@@ -577,6 +578,8 @@ textarea {
   border-radius: 16px;
   padding-bottom: 20px;
   display: none;
+  overflow: hidden;
+  overflow-x: scroll;
```

Comment:
- [ ] Please 🙏🏽 correct the background dimension to be similar to the Figma template.

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/220162696-6b97d6c7-e9de-46ed-a702-6dc1f7111bbd.png) | ![image](https://user-images.githubusercontent.com/58642949/220162519-77d86ef6-599b-41a0-8c1e-16f28bfe6ea3.png) |

Reply:
hi @franclobo 
please check again ,I managed to fix the hamburger button

Reply:
hi @franclobo
please check again,I managed to fix the hamburger button